src/components/MintDialog/pages/Bridge/NotStarted.tsx

```diff
@@ -92,7 +95,18 @@ export const NotStarted: FC<NotStartedProps> = ({
         </div>
         <div className="flex flex-col gap-4">
           {wrongChain && switchNetwork ? (
-            <Button onClick={() => switchNetwork(l1.id)}>Switch to L1</Button>
+            <Button
+              onClick={() => {
+                const isCoinbaseBrowser = getIsCoinbaseBrowser()
```

Comment:
I think we want to auto do this? So this should be like in a useEffect on page load here? Don't want to have to click if coinbase wallet

Reply:
Right - I only had this in here because it was weird in metamask

I'll update this